system/jlib/jdebug.hpp

```diff
@@ -123,28 +123,20 @@ struct UserSystemTime_t
 
 interface ITimeReportInfo
 {
-    virtual void report(const char * scope, const char * description, const __int64 totaltime, const __int64 maxtime, const unsigned count) = 0;
+    virtual void report(const char * scope, const __int64 totaltime, const __int64 maxtime, const unsigned count) = 0;
 };
 class StringBuffer;
 class MemoryBuffer;
 struct ITimeReporter : public IInterface
 {
   virtual void addTiming(const char * scope, cycle_t cycles) = 0;
   virtual void mergeTiming(const char * scope, cycle_t totalcycles, cycle_t maxcycles, const unsigned count) = 0;
-  virtual unsigned numSections() = 0;
-  virtual __int64 getTime(unsigned idx) = 0;
-  virtual __int64 getMaxTime(unsigned idx) = 0;
-  virtual unsigned getCount(unsigned idx) = 0;
-  virtual StatisticKind getTimerType(unsigned idx) = 0;
-  virtual StatisticScopeType getScopeType(unsigned idx) = 0;
-  virtual StringBuffer &getScope(unsigned idx, StringBuffer &s) = 0;
   virtual StringBuffer &getTimings(StringBuffer &s) = 0;
   virtual void printTimings() = 0;
   virtual void reset() = 0;
-  virtual void mergeInto(ITimeReporter &other) = 0;
+  virtual void mergeInto(ITimeReporter &other) = 0; // Could be removed - not actually used.
```

Comment:
Perhaps this would be a good point to do so?

Reply:
Actually the comment isn't right - it is used internally within the implementation.  I will change the comment.